content/modules/redisgears/write-behind.md

```diff
@@ -0,0 +1,125 @@
+---
+Title: Write-Behind Caching
+description:
+weight: 50
+alwaysopen: false
+categories: ["Modules"]
+---
+
+## Write-Behind Caching Defined
+
+Write-behind caching is a caching strategy in which the cache layer itself connects to the backing database. This means that your applications need only ever connect to your cache layer, and the cache then updates the backing database as needed. Redis Labs currently supports write-behind caching for both Redis Cloud and Redis Enterprise.
+
+Here's how these caching patterns work:
+
+1. Your application uses the cache for reads and writes.
+1. The cache syncs any changed data to the backing database asynchronously.
+
+<!-- ### Read-Through Caching
+
+1. Your application reads from the cache. If there's a cache hit, the data is returned.
+1. If there's cache miss, the cache retrieves the data from your backing database (think Oracle, PostgreSQL, etc.). The data is then stored in the cache and returned. -->
+
+## Write-Behind Caching with RedisGears
+
+[RGSync](https://github.com/RedisGears/rgsync) is a RedisGears recipe that uses the write-behind pattern to synchronize Redis data structures to a backing data store (Oracle, MySQL, Cassandra, and Snowflake are currently supported).
```

Comment:
Eventually (hopefully next week) this should additionally point to the download centre as well